mcs/class/System/ReferenceSources/Win32Native.cs

```diff
@@ -0,0 +1,1790 @@
+using System.IO;
+
+namespace Microsoft.Win32
+{
+	static class Win32Native
```

Comment:
It looks like the constants can be local to the method with switch and save another huge class with fields space